Sources/PitchSpeller/Data Structures/Swappable.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Swappable.swift
+//  PitchSpeller
+//
+//  Created by Benjamin Wetherfield on 6/30/18.
+//
+
+protocol Swappable: Symmetric {
+    var elput: (A,A) { get }
```

Comment:
As above, maybe only `swapped` is required here, and `elput` could be an implementation detail downstream.